src/org/thoughtcrime/securesms/notifications/SingleRecipientNotificationBuilder.java

```diff
@@ -321,4 +320,14 @@ private CharSequence getBigText(List<CharSequence> messageBodies) {
     return content;
   }
 
+  private static Drawable obtainAvatarPlaceholderDrawable(Context context, Recipient recipient) {
```

Comment:
What about group notifications?

Reply:
Whoops, looks like I forgot about that one.